packages-next/admin-ui/src/system/createAdminUIServer.ts

```diff
@@ -11,14 +10,16 @@ export const createAdminUIServer = async (
   projectAdminPath: string,
   sessionStrategy?: SessionStrategy<any>
 ) => {
+  const thing = 'n' + 'ext';
```

Comment:
Probs worth adding a comment above this and the other one to say this is to stop webpack from bundling next inside neext.

Reply:
Is the string concat actually needed here? Is that thing that will _trick_ webpack?

Reply:
Don't think so since it works on the other one without the string concat. Having the string in a variable and then calling require with it is def needed though.

Reply:
Yah cool, will just put it in a var and dump the concat then, cheers!